lib/dradis/plugins/thor_helper.rb

```diff
@@ -2,9 +2,28 @@ module Dradis
   module Plugins
     # Helper methods for plugin Thor tasks
     module ThorHelper
+      attr_accessor :task_options, :logger
+
       def detect_and_set_project_scope
-        ;
+        task_options[:project_id] = ENV['PROJECT_ID'].to_i if ENV['PROJECT_ID']
+      end
+
+      def task_options
+        @task_options ||= {logger: logger}
+      end
+
+      def logger
```

Comment:
when the logger is STDOUT as it's always the case with Thor invokation, do we actually need to close it?

I think it's weird for this method to be able to use `logger` freely without having to create the object but then being responsible for closing it:

https://github.com/dradis/dradis-projects/blob/master/lib/tasks/thorfile.rb#L41-L42